openwisp_ipam/tests/test_users_integration.py

```diff
@@ -1,11 +0,0 @@
-from openwisp_users.tests.test_admin import TestUsersAdmin
```

Comment:
why is this removed?

Reply:
Commented it out during testing, forgot to add it back, thanks for pointing it out.
Updated.